androidApp/src/main/kotlin/com/intive/picover/profile/view/ProfileActions.kt

```diff
@@ -21,21 +21,15 @@ import dev.icerock.moko.resources.compose.stringResource
 fun ProfileActions(onLogoutClick: () -> Unit, onDeleteAccountCLick: () -> Unit) {
 	val context = LocalContext.current
 	val items = listOf(
-		ProfileActionsItem.Licenses(
-			onClick = {
-				LibsBuilder().start(context)
-			},
-		),
-		ProfileActionsItem.Logout(onClick = onLogoutClick),
-		ProfileActionsItem.DeleteAccount(onClick = onDeleteAccountCLick),
-		ProfileActionsItem.GitHub(
-			onClick = {
-				CustomTabsIntent
-					.Builder()
-					.build()
-					.launchUrl(context, Uri.parse(context.getString(R.string.GitHub)))
-			},
-		),
+		ProfileActionsItem.ShowLicenses to { LibsBuilder().start(context) },
+		ProfileActionsItem.Logout to onLogoutClick,
+		ProfileActionsItem.DeleteAccount to onDeleteAccountCLick,
+		ProfileActionsItem.ShowGitHub to {
+			CustomTabsIntent
+				.Builder()
+				.build()
+				.launchUrl(context, Uri.parse(context.getString(R.string.GitHub)))
+		},
 	)
 	LazyColumn {
 		items(items) {
```

Comment:
Some adjustments are missing in this part? 😄

Reply:
That's the result of making PR's in hurry 😄